### ISSUE TITLE
[a11y]: ai-chat does not meet color contrast ratio for interactive message buttons

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#863]
### Browser

Chrome

### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

WCAG Mapping: 1.4.3 Contrast Minimum

### Description

When the Rate a Response dialog is opened buttons in the dialog fail the minimum contrast ratio after activation
After the agent activation, few selection buttons appears where CC (color contrast) Fails for the buttons to meet contrast ratio 4.5:1 both in normal and hover states.
Note: with the background gradient in the bottom of page the buttons fails for CC.

Recommended fix: Ensure the button text and background colors (for both normal and hover states) meet the minimum contrast ratio of 4.5:1 as required by accessibility standards.

User impact: Users with visual impairments may not be able to read the button labels clearly. This can lead to confusion and difficulty.

<img width="544" height="340" alt="Image" src="https://github.com/user-attachments/assets/2417f910-6fd7-4022-b2b8-8e6e2b1c3935" />

### Reproduction/example

n/a

### Steps to reproduce

1. add interactive buttons to message
2. scroll the message with the buttons to the bottom s they overlap with the background gradient
3. verify the contrast values for the buttons

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
